metric-anomaly-task-manager/src/main/resources/configs/common/application.conf

```diff
@@ -12,7 +12,7 @@ job.config = {
   tenant_id = ${?DEFAULT_TENANT_ID}
 }
 
-ruleSource = {
+alertRuleSource = {
```

Comment:
handle this in the code as well

Reply:
@sarthak77 this isn't done yet

Reply:
where exactly in the code?

Reply:
just see were this config is being used, maybe AlertTaskConverter or somewhere